### PR TITLE
fixing RMCform in edit site

### PR DESCRIPTION
**Pull request by Frankkwizera**

### CODE REVIEW

File: seshdash/static/sesh-ui/js/sesh_dashboard.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 
 var csrftoken = getCookie('csrftoken');
 
+if ($('#id_api_key').length > 0) {
```
Comment (jlark) [inline seshdash/static/sesh-ui/js/sesh_dashboard.js]: ahh I don't like this :-1:  isn't it possible to do this with the formset?

Comment (Frankkwizera) [inline seshdash/static/sesh-ui/js/sesh_dashboard.js]: my bad,,didn't know how formset works!

Comment (Frankkwizera) [inline seshdash/static/sesh-ui/js/sesh_dashboard.js]: let me switch

**Comment by jlark** [comment]
LGTM

**Pull request merged**